utils/streams.js

```diff
@@ -0,0 +1,136 @@
+(function () {
+	const fs = require('fs');
+	const through = require('through2');
+	const csv = require('csvtojson');
+	const async = require('async');
+	const path = require('path');
+	const request = require('request');
+
+	const HELP = 'help';
+	const ACTION ='action';
+	const FILE ='file';
+	const PATH ='path';
+
+	const CSS_PATH = 'https://www.epam.com/etc/clientlibs/foundation/main.min.fc69c13add6eae57cd247a91c7e26a15.css';
+
+	let argv = require('minimist')(process.argv);
```

Comment:
Consider configuring aliases via `alias` option of `minimist`.

Reply:
done